mungegithub/mungers/approvers/owners.go

```diff
@@ -188,9 +217,8 @@ type Approvers struct {
 // UnapprovedFiles returns owners files that still need approval
 func (ap Approvers) UnapprovedFiles() sets.String {
 	unapproved := sets.NewString()
-	for fn := range ap.Owners.GetOwnersSet() {
-		approvers := ap.Owners.repo.Approvers(fn)
-		if approvers.Intersection(ap.Approvers).Len() == 0 {
+	for fn, approvers := range ap.Owners.GetApprovalStatus(ap.Approvers) {
+		if len(approvers) == 0 {
```

Comment:
we're really overloading the approvers term.  It's a struct, a field within a struct and a commonly used local variable.  Can we use a different var name?